app/models/UserAnswers.scala

```diff
@@ -30,7 +30,7 @@ final case class UserAnswers(
   tasks: Map[String, TaskStatus] = Map(),
   status: SubmissionState.Value,
   departureId: Option[String] = None
-) {
+) extends UserAnswersResponse {
```

Comment:
Ideally UserAnswersResponse would be a sealed trait, and we'd have some case class containing the user answers